.travis.yml

```diff
@@ -11,7 +11,13 @@ sudo: false
 dist: trusty
 services:
   - docker
-language: generic
+language:
```

Comment:
This whole set of changes is not needed. Gareth made a script that does this for us

Reply:
Removed the set of changes.